torch/csrc/jit/tensorexpr/ir.h

```diff
@@ -274,6 +274,22 @@ class Let : public ExprNode<Let> {
   Expr body_;
 };
 
+class ExprStmt : public StmtNode<ExprStmt> {
```

Comment:
Minor nit: LLVM calls this StmtExpr. 

Since we are moving closer to LLVM conventions, might as well just use that. Add a TODO to refactor this.

https://clang.llvm.org/doxygen/classclang_1_1StmtExpr.html

Reply:
I think this is actually the other way around - I need to wrap an Expr in a Stmt to eval() it.  The StmtExpr is wrapping a Stmt in an Expr.